### ISSUE TITLE
Stackblitz examples in the Rxjs introduction seem broken

### ISSUE THREAD

**Issue by ChuckIrvine** [ReactiveX/rxjs#4605]
At the RxJS introduction website (https://rxjs-dev.firebaseapp.com/guide/overview), the StackBlitz examples aren't working, at least the first several that I tried. The first four examples on that page give:

```
Error occurred:
null is not an object (evaluating 'button.addEventListener')
```

```
Error occurred:
Can't find variable: rxjs
```

```
Error occurred:
null is not an object (evaluating 'button.addEventListener')
```

```
Error occurred:
Can't find variable: rxjs
```

**Comment by jgbpercy** [comment]
The html files for all these examples seem to be empty...

**Comment by ChuckIrvine** [comment]
Correct. I should have mentioned that. Thanks

**Comment by JWO719** [comment]
Hi thanks for spotting this. As we merged the Stackblitz integration we knew that maybe some of the examples are broken. This might occur due to missing import statements or similiar minor things. I really appreciate any contribution for those examples. Do you maybe want to fix those? If you need any help with that, just let me know!

**Comment by ChuckIrvine** [comment]
I'm afraid my level of knowledge isn't yet sufficient. I was looking at the examples in an attempt to better understand the material, as would probably a lot of your new folks.

**Comment by santoshyadav198613** [comment]
Hi @JWO719,
I can take this up, let me know if this is open.

**Comment by JWO719** [comment]
I'm not aware that anyone already started working on that, so feel free to pick that up. If you need any help, just let me know

**Comment by santoshyadav198613** [comment]
Hi @JWO719 ,
Can you point me to the documentation.

**Comment by santoshyadav198613** [comment]
Also there is an open PR for stackblitz https://github.com/ReactiveX/rxjs/issues/4621
Looks like i have to wait till it gets sorted.

**Comment by BioPhoton** [comment]
Yes, there is a PR. It should solve all code format problems related to stackblitz.
Also, all snippets are now refactored to TypeScript.

**Comment by JWO719** [comment]
closed by #4689
